src/pcp/js/widgets.js

```diff
@@ -0,0 +1,304 @@
+/*
+
+Copyright 2013-2017 OCAD University
+
+
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+
+BSD license. You may not use this file except in compliance with one these
+
+Licenses.
+
+
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+
+*/
+
+/* global fluid */
+
+"use strict";
+(function (fluid) {
+    var gpii = fluid.registerNamespace("gpii");
+
+    fluid.registerNamespace("gpii.pcp.widgets");
+
+    gpii.pcp.widgets.noop = function () {
```

Comment:
We can axe this - if you need a general function that does nothing you can use fluid.identity - but it looks like the only use is as a click binder in an invoker, in which case you can use fluid.notImplemented since I imagine you want to force the user to override this